packages/client/src/rtc/Call.ts

```diff
@@ -597,6 +599,28 @@ export class Call {
     });
 
     try {
+      const clientDetails: ClientDetails = {};
+      if (typeof navigator === 'undefined') {
+        // TODO RN
```

Comment:
I'm creating an task for this, we should lean here on our own framework-specific APIs as much as possible i.e. [Platform](https://reactnative.dev/docs/platform-specific-code#platform-module) and [DeviceInfo](https://github.com/react-native-device-info/react-native-device-info)

Reply:
btw navigator is not undefined in RN

Reply:
@santhoshvai can you help me with the adequate checks for this use-case?

Reply:
I've seen this in another PR: https://github.com/GetStream/stream-video-js/pull/395/files#diff-61771ffe92cbe5568b7ceecac9671fc8090fc07df09ad2ff8a511f842a5320d0 - should I be using this?

Reply:
Fixed, please check if this looks ok

Reply:
Correct ✅

Reply:
Thanks for checking

Reply:
missed reading this @szuperaz glad that you found it!